templates/openstack/what-is-openstack.html

```diff
@@ -10,446 +12,565 @@
-        </p>
-      </article>
-      <article class="col-6 p-card">
-        <div class="p-card__title">
-          <img src="https://assets.ubuntu.com/v1/dc69e636-firmus-black.png"
-               width="200"
-               style="margin-bottom: 6px"
-               alt="" />
-        </div>
-        <hr class="p-rule--muted" />
-        <p class="p-card__content">
-          In 2022, Firmus &mdash; Australian cloud infrastructure provider &mdash; announced that it has built an ultra-efficient and sustainable public cloud using an innovative immersion cooling solution and open-source technologies. Those included Canonical's <a href="/openstack/features">Charmed OpenStack</a> and <a href="/kubernetes/charmed-k8s">Charmed Kubernetes</a>.
-        </p>
-        <br />
-        <p>
-          <a href="/engage/kubernetes-openstack-case-study-firmus">Read the case study&nbsp;&rsaquo;</a>
-        </p>
-      </article>
-    </div>
-    <div class="row u-equal-height">
-      <article class="col-6 p-card">
-        <div class="p-card__title">
-          <img src="https://assets.ubuntu.com/v1/01d8505d-pacific-textiles.jpg"
-               alt="" />
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_1' -%}
+      {{ image(url="https://assets.ubuntu.com/v1/1155d299-kadokawa-logo.png",
+        alt="Kadokawa",
+        width="852",
+        height="82",
+        hi_def=True,
+        loading="lazy") |safe 
+      }}
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_1' -%}
+      <p>
+        In 2023, KADOKAWA Connected &mdash; a subsidiary of leading Japanese media conglomerate KADOKAWA Corporation, delivering digital transformation, infrastructure development, and ICT services to the wider KADOKAWA Group &mdash; engaged Canonical to build a new, open-source private cloud as a replacement for their legacy proprietary virtualization platform.
+      </p>
+      <div class="p-cta-block">
+        <a href="/engage/kadokawa-connected-case-study-charmed-openstack">Read the case study&nbsp;&rsaquo;</a>
+      </div>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_2' -%}
+      {{ image(url="https://assets.ubuntu.com/v1/fdc33102-logo-2.png",
+        alt="Firmus",
+        width="852",
+        height="82",
+        hi_def=True,
+        loading="lazy") |safe 
+      }}
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_2' -%}
+      <p>
+        In 2022, Firmus &mdash; Australian cloud infrastructure provider &mdash; announced that it has built an ultra-efficient and sustainable public cloud using an innovative immersion cooling solution and open-source technologies. Those included Canonical's <a href="/openstack/features">Charmed OpenStack</a> and <a href="/kubernetes/charmed-k8s">Charmed Kubernetes</a>.
+      </p>
+      <div class="p-cta-block">
+        <a href="/engage/kubernetes-openstack-case-study-firmus">Read the case study&nbsp;&rsaquo;</a>
```

Comment:
Let's do this for all the CTAs here, otherwise these links are indistinguishable from one another when you tab through them
```suggestion
        <a href="/engage/kubernetes-openstack-case-study-firmus" aria-label="Read the Firmus case study">Read the case study&nbsp;&rsaquo;</a>
```